src/OrchardCore.Modules/OrchardCore.Spatial/Fields/GeoPointField.cs

```diff
@@ -0,0 +1,11 @@
+using OrchardCore.ContentManagement;
+
+namespace OrchardCore.Spatial.Fields
+{
+    public class GeoPointField : ContentField
+    {
+        public double? Latitude { get; set; }
```

Comment:
Does it have to be nullable?

Reply:
Yes, because when you edit a content item you don't necesarly enter a Geopoint for all of them. So, just like all other fields we allow them to be nullable.

Reply:
Or maybe we need a GeoPointFieldViewModel which is nullable and then a GeoPointField which is not.